packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -151,17 +130,26 @@ const handleResource = (
       ...result,
     }
 
-    if (
-      props._key &&
-      !lodash.isEqual(newResource, resourceMap.get(props._key))
-    ) {
-      resourceMap.set(props._key, newResource)
-      setResources([...resourceMap.values()])
+    if (!lodash.isEqual(newResource, resourceMap.get(trueKey))) {
+      resourceMap.set(trueKey, newResource)
+      // TODO Do we need this? It's causing infinite loops
+      // setResources([...resourceMap.values()])
```

Comment:
I think this is for nested resources... maybe @johno remember? Anyways, we'll figure it out again when we return to that.

Reply:
I think at one point it was for nested resources, but the code that was leveraging it is long gone IIRC.